packages/suite/src/actions/suite/__fixtures__/metadataActions.ts

```diff
@@ -53,18 +53,31 @@ const setDeviceMetadataKey = [
                     metadata: { status: 'cancelled' },
                 },
             },
+            {
+                type: METADATA.DISABLE,
+            },
+        ],
+    },
+    {
+        description: `Master key successfully generated`,
```

Comment:
Is this message going to be displayed to the user? If yes, we should localize it.

Reply:
No, this is test case decription. it(this string goes here, testFnCallback)

Reply:
Indeed, I was reviewing going upwards and missed the file title :smile: All good